intermediate_source/torch_compile_full_example.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+
+"""
+``torch.compile`` End-to-End Tutorial
+=================================
+**Author:** William Wen
+"""
+
+import warnings
```

Comment:
Do you want this showing at the top like it is? It doesn't make much sense in the flow of the page.